pkg/apis/operator/v1alpha1/types_garden.go

```diff
@@ -126,19 +126,15 @@ type DNSDomain struct {
 
 // RuntimeNetworking defines the networking configuration of the runtime cluster.
 type RuntimeNetworking struct {
-	// Nodes is the CIDR of the node network. This field is immutable.
-	// +kubebuilder:validation:MinLength=1
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	// Nodes are the CIDRs of the node network. Elements can be appended to this list, but not removed.
 	// +optional
-	Nodes *string `json:"nodes,omitempty"`
-	// Pods is the CIDR of the pod network. This field is immutable.
-	// +kubebuilder:validation:MinLength=1
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
-	Pods string `json:"pods"`
-	// Services is the CIDR of the service network. This field is immutable.
-	// +kubebuilder:validation:MinLength=1
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
-	Services string `json:"services"`
+	Nodes []string `json:"nodes,omitempty"`
+	// Pods are the CIDRs of the pod network. Elements can be appended to this list, but not removed.
+	// +kubebuilder:validation:MinItems=1
```

Comment:
```suggestion
	// +kubebuilder:validation:MinItems=1
	// +kubebuilder:validation:MaxItems=2
```

Does this make sense? Same for other slices.

Reply:
More or less answered in https://github.com/gardener/gardener/pull/11251#discussion_r1937322692.

TLDR: I would prefer to only validate the minimum size for the runtime cluster ranges, but agree that a limit of 2 makes sense for the virtual cluster service range.